src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/AndroidUpdateResourcesTest.cs

```diff
@@ -26,7 +26,7 @@ public void CheckProjectReferenceAlias ()
 			var proj = new XamarinAndroidApplicationProject () {
 				References = {
 					new BuildItem.ProjectReference (Path.Combine("..", library.ProjectName, Path.GetFileName (library.ProjectFilePath)), "Library1") {
-						Metadata = { { "Aliases", "Lib1" } },
+						Metadata = { { "Aliases", "Lib1, LibA1,LibB1" } },
```

Comment:
Are aliases always comma-delimited? Or can they be `;`, too?

I feel like I have barely ever used this feature of C#... 👀

Reply:
The documentation is.... not specific :(
```
Aliases | Optional string. Any aliases for the reference.
```

https://docs.microsoft.com/en-us/visualstudio/msbuild/common-msbuild-project-items?view=vs-2019

Reply:
I will try to see if `;` works. If it does we can use that too

Reply:
it doesn't 

```error MSB6001: Invalid command line switch for "csc.exe". System.ArgumentException: MSB3053: The assembly alias "LibA1;LibB1"```

Reply:
So MSbuild will not parse that property out if you use `;` . Only if you use `,`